src/main/java/org/ays/emergency_application/model/request/EmergencyEvacuationApplicationRequest.java

```diff
@@ -76,22 +74,22 @@ public class EmergencyEvacuationApplicationRequest {
     @Valid
     private AysPhoneNumberRequest applicantPhoneNumber;
 
-
     @JsonIgnore
     @AssertTrue(message = "all applicant fields must be filled")
     @SuppressWarnings("This method is unused by the application directly but Spring is using it in the background.")
     private boolean isAllApplicantFieldsFilled() {
 
-        if (StringUtils.isEmpty(this.applicantFirstName) && StringUtils.isEmpty(this.applicantLastName) && this.applicantPhoneNumber == null) {
+        if (StringUtils.isEmpty(this.applicantFirstName) &&
+                StringUtils.isEmpty(this.applicantLastName) && this.applicantPhoneNumber == null) {
             return true;
         }
 
-
         return !StringUtils.isBlank(this.applicantFirstName) && !StringUtils.isBlank(this.applicantLastName)
                 &&
                 this.applicantPhoneNumber != null
                 &&
-                !(StringUtils.isBlank(this.applicantPhoneNumber.getCountryCode()) && StringUtils.isBlank(this.applicantPhoneNumber.getLineNumber()));
+                !(StringUtils.isBlank(this.applicantPhoneNumber.getCountryCode()) &&
+                        StringUtils.isBlank(this.applicantPhoneNumber.getLineNumber()));
```

Comment:
Burayı da eski hâline alabilir miyiz?
```suggestion
                !(StringUtils.isBlank(this.applicantPhoneNumber.getCountryCode()) && StringUtils.isBlank(this.applicantPhoneNumber.getLineNumber()));
```